be/src/env/env_posix.cpp

```diff
@@ -498,6 +499,11 @@ class PosixEnv : public Env {
         return Status::OK();
     }
 
+    Status new_random_access_file(const std::string& fname,
```

Comment:
How about change the name to make if different with another `new_random_access_file()` method?

Reply:
I think it is just ok, because of overload is common.

Reply:
Without documentation or a clear name, user will consider the following

```
std::shared_ptr<RandomAccessFile> f;
env->new_random_access_file(filename, &f);
```

to be the same as 

```
std::unique_ptr<RandomAccessFile> f1;
env->new_random_access_file(filename, &f1);
// upgrade to shared_ptr
std::shared_ptr<RandomAccessFile> f = std::move(f1);
```

which is not true.

Reply:
ok, I will add some comments.